content/handbook/product/product-processes/planning-with-gitlab.md

```
@@ -0,0 +1,155 @@
+---
+
+私たちは GitLab を使って製品戦略をドキュメント化し、バックログを管理しています。このプロセスのカギとなるコンセプトは次のとおりです:
+
+- [マイルストーン](https://docs.gitlab.com/user/project/milestones/): 当社の [製品リリース](https://about.gitlab.com/releases/) と整合し、グループの計画タイムボックスとして使用します。
+- [Issue](https://docs.gitlab.com/user/project/issues/): 単一のマイルストーン内で提供できるユーザー価値の単位を捉えます。
+- [タスク](https://docs.gitlab.com/user/tasks/)（オプション）: Issue を、より詳細な実装ステップに分解します。
+- [エピック](https://docs.gitlab.com/user/group/epics/): 関連する Issue をテーマや目標にグループ化します。エピックを永続的なコンテナにするのではなく、具体的な作業範囲を表し、作業が完了したらエピックをクローズできることを目標とするのがベストプラクティスです。
+- [ボード](https://docs.gitlab.com/user/project/issue_board/): [製品開発フロー](/handbook/product-development/how-we-work/product-development-flow/_index.md) を進行する作業を可視化し、マイルストーン計画に役立ちます。
+- [ロードマップ](https://docs.gitlab.com/user/group/roadmap/): エピックをタイムラインビューで可視化するのに役立ちます。
+
+### Issue
```

markdownlint-cli2 (0.22.1)

[warning] Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)